src/components/CategoryFilter.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { bookCategories } from '../utils';
+
+
+const filterCategories = [...['All'], ...bookCategories];
```

Comment:
We overdid the nesting of 'All`. We could have simply:

```javascript
['All', ...bookCategories]
```
Without doing `...['All]`.

Another thing, can we just place this variable right inside the component instead of hanging it outside as a global variable? Anyway it is in `CategoryFilter` that we are using it.

Reply:
### Kindly implement this change. This may appear trivial/insignificant but it's important in JavaScript to understand the basic usage of the rest and spread operators.